sysutils/stressdrive/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+PortGroup           xcode 1.0
+
+github.setup        rentzsch stressdrive 1.3.2
+categories          sysutils
+platforms           darwin
+maintainers         @toy
+license             MIT
+
+description         Tool meant to verify correct operation of a drive
+long_description    Tool meant to verify correct operation of a drive by \
```

Comment:
this can re-use the description as in:
`long_description ${description} by.....`